src/toxcore/dht/server/mod.rs

```diff
@@ -604,6 +609,25 @@ impl Server {
 
     /// actual send method
     fn send_to(&self, addr: SocketAddr, packet: DhtPacket) -> IoFuture<()> {
+        if !self.is_ipv6_mode { // DHT node is running in ipv4 mode
```

Comment:
Please don't do `if !self.is_ipv6_mode`. Better code it like: `if self.is_ipv6_mode {  // DHT node is running in ipv6 mode }`

Reply:
done